tests/test_manifest.cpp

```
@@ -1839,11 +2013,1085 @@ TEST(ManifestRevisionTest, SchemaEmittedBeforeRevisionInHeader)
+            rec.label = "export_poison";
+            rec.kind = an::AnchorKind::ExportName;
+            rec.export_name = std::string(poison);
+            record_rejects(rec, "record.export_name");
+        }
+        {
+            an::Anchor source;
+            source.label = "adopt_module";
+            source.kind = an::AnchorKind::ExportName;
+            source.export_name = "Symbol";
+            source.export_module = std::string(poison);
+            anchor_rejects(source, "anchor.export_module");
```

Cppcheck (2.21.0)

[error] Using object that is a temporary.

(danglingTemporaryLifetime)